presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -310,6 +310,56 @@ Statistical Aggregate Functions
 
     Returns the sample covariance of input values.
 
+.. function:: differential_entropy(bucket_count, sample, weight, "fixed_histogram_mle", min, max) -> double
+
+    Returns the approximate log-2 differential entropy from a random variable's sample outcomes. Broadly speaking,
+    the function internally creates a conceptual histogram of the sample values, calculates the counts, and
+    then approximates the entropy using maximum-likelihood estimation.
+
+    The parameter `bucket_count` determines the number of histogram buckets. The parameter `min` is the minimal value
+    of a sample - all values smaller than `min`, will be made into `min`. The parameter `max` is the minimal value
+    of a sample - all values larger than `max`, will be made into `max`. The parameter `weight` is the weight
+    of the sample, and must be non-negative.
+
+    See more in [Beirlant2001]_.
+
+.. function:: differential_entropy(bucket_count, sample, weight, "fixed_histogram_jacknife", min, max) -> double
```

Comment:
More broadly, I wonder if it's better to represent these different strategies with different names.  There's no analog right now for using a string to specialize the internal detail of the function.

Reply:
Answered in comment below.